docker/amdvlk.Dockerfile

```diff
@@ -51,7 +51,12 @@ WORKDIR /vulkandriver
 RUN repo init -u https://github.com/GPUOpen-Drivers/AMDVLK.git -b "$BRANCH" \
     && repo sync -c --no-clone-bundle -j$(nproc) \
     && cd /vulkandriver/drivers/spvgen/external \
-    && python2 fetch_external_sources.py
+    && python2 fetch_external_sources.py \
+    && if echo "$FEATURES" | grep -q "+sanitizer" ; then \
+         cd glslang \
+         && curl https://gist.githubusercontent.com/Flakebi/530ff13056407237fdbdeaab5dd740bf/raw/43e0e2dc167a2a87f0a507acc01e83edd2515118/glslang.patch > glslang.patch \
+         && patch -p1 < glslang.patch; \
```

Comment:
This is a rather ugly hack. Would it be possible to fix glslang first before submitting this PR?
I'm mainly afraid that the patch will eventually stop applying.

Reply:
We could use `git checkout <some more recent version of glslang>` instead, that wouldn’t break so easily.
Otherwise we have to wait until the pinned version is updated.

Reply:
This sound like a good plan to me. If updating the pinned version will take more than a few days, then I'd go ahead with `git checkout`.